README.md

```diff
@@ -1,3 +1,123 @@
 # konan-housing-model
 
-A Konan-ready ML model, packaged and distributed as a Docker image, to estimate the sale price of house based on the popular Ames Housing Dataset
+[Konan](https://konan.ai) is your one-stop platform to bring your AI models to life. Typically, one would be required to do some tedious boring work in order to prepare the machine learning or data science model for deployment. Not with us! With **Konan**, gone are the days of worrying about a model's *dependencies*, battling with *containerization* or obsessively hand-holding the *infrastructure*. This repo will guide you in preparing your machine learning model in as seamless a manner as possible! Of course, don't forget to also checkout **Konan's** rich [documentation](https://docs.konan.ai) for additional info.
```

Comment:
I would check with Heba what intro should go there so that it aligns with the brand marketing we have on other channels.

Reply:
I contacted Heba and waiting on more input from her.